tests/cnt-i18n_xhtml-xml-lang/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,12 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="fr">
```

Comment:
I would call this cnt-xml-lang or cnt-xhtml-xml-lang (but IMO that's redundant; XHTML is XML).